packages/services/tokens/src/environment.ts

```diff
@@ -10,6 +10,14 @@ const numberFromNumberOrNumberString = (input: unknown): number | undefined => {
 
 const NumberFromString = zod.preprocess(numberFromNumberOrNumberString, zod.number().min(1));
 
+const BooleanFromString = zod.preprocess(val => {
+  if (typeof val === 'string') {
+    if (['1', 'true'].includes(val.toLowerCase())) return true;
+    if (['0', 'false'].includes(val.toLowerCase())) return false;
+  }
+  return val;
+}, zod.coerce.boolean());
```

Comment:
We already have introduced a way of handling boolean fields e.g. here:
https://github.com/graphql-hive/platform/blob/94998dd8e3df1107c1bdf6c1f22eac91358e35d7/packages/services/server/src/environment.ts#L42

Let's use this way also for the `REDIS_TLS_ENABLED` environment variable and keep things simply without coercion.

Reply:
fixed